lib/console_web/router.ex

```diff
@@ -66,6 +66,8 @@ defmodule ConsoleWeb.Router do
     pipe_through ConsoleWeb.RouterApiPipeline
```

Comment:
Why is the scope here under ConsoleWeb.Router and not just ConsoleWeb?

Reply:
because router has a different authentication scheme

Reply:
and can access data and routes that a regular user cannot

Reply:
Is there a separate file for ConsoleWeb.Router? I get the RouterApiPipeline is different than AuthApiPipeline but dont' get why I can't hit the router through postman when scope "/api/router" is under ConsoleWeb.Router and not ConsoleWeb

Reply:
nvm lol i figured it out with your later comments